src/requests/getForecast.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+import axios from "axios";
+
+const getForecast = (
+  searchText,
+  setSelectedDate,
+  setForecasts,
+  setLocation,
+  setErrorMessage
+) => {
+  let endpoint = "https://mcr-codes-weather-app.herokuapp.com/forecast";
+  if (searchText) {
+    endpoint += `?city=${searchText}`;
+  }
+  axios
+    .get(endpoint)
+    .then((response) => {
+      setSelectedDate(response.data.forecasts[0].date);
+      setForecasts(response.data.forecasts);
+      setLocation(response.data.location);
+    })
```

Comment:
You'd also need to reset the error message state here 
```suggestion
    .then((response) => {
      setErrorMessage("");
      setSelectedDate(response.data.forecasts[0].date);
      setForecasts(response.data.forecasts);
      setLocation(response.data.location);
    })
```